synapse/lib/ast.py

```diff
@@ -4177,17 +4177,19 @@ async def run(self, runt, genr):
                 # runt node property permissions are enforced by the callback
                 runt.confirmPropSet(prop)
 
+            isndef = isinstance(prop.type, s_types.Ndef)
             isarray = isinstance(prop.type, s_types.Array)
```

Comment:
Do we want to support setting ndef arrays like this as well?

Reply:
Probably yep. Especially with some of the updates coming in a subsequent PR. Might be easiest to implement tests along with those model updates to support the idea.

Reply:
Sounds good, we don't currently have any ndef array props in the model so seems reasonable to wait for that PR to add it in.